README.md

```diff
@@ -0,0 +1,152 @@
+# Supported LXC Containers, by Symas Corp.
+
+## What's here?
+
+[OpenLDAP](https://openldap.org) and related software, provided by
+[Symas](https://symas.com), containerized and ready for production use.
+
+
+## How to contact Symas for commercial support
+
+If you've not tried running your container with `SYMAS_DEBUG` set to `true` then you might try that first, this container is primed to give you the information you need to debug issues yourself.  Also, please take a look at our [knowledge base](https://kb.symas.com/) if going to the [OpenLDAP site](https://openldap.org/) and reading the [documentation](https://openldap.org/doc/), the [quick start guide](https://openldap.org/doc/admin26/quickstart.html), and the detailed [manual pages](https://openldap.org/software/man.cgi) didn't help.  [What we publish](https://repo.symas.com) is what we provide inside the [containers](https://github.com/symas/containers) we provide to you, everything is open-source.
+
+Of course, you can always call us at: +1.650.963.7601 or email [sales](mailto:sales@symas.com) or [support](mailto:support@symas.com) teams directly with questions.  More on our support offerings can be [found on our website](https://www.symas.com/symas-tech-support).  We're also available on the Symas Discord [#openldap](https://discord.gg/t6upYQDx2) channel and chat with us directly.
+
+Reach out to us, we're here to help.
+
+
+## Why use our images?
+
+* Symas has, for over a decade, built, maintained, and commercially supported
+  the OpenLDAP codebase.
+* All our work on OpenLDAP has always been, and will always be open-source.  We
+  are a [commercial support company](mailto:support@symas.com), here if and when
+  you need us.
+* We commit to promptly publish new versions of this chart for every release of
+  the software going forward.
+* Our images contain the latest bug fixes and features released, not just in OpenLDAP
+  but in supporting libraries.
+* We've based our containers on the amazing [Bitnami
+  containers](https://github.com/bitnami/containers/) existing work.
+* We'd like to thank them for open sourcing their work allowing the community to
+  benefit.  Our shared philosophy makes us stronger.
+* Because we are using the Bitnami container infrastructure and scripts this is
+  a drop-in replacement for any existing use of the `bitnami/openldap`
+  containers sharing the same version.
+* We base our images on Bitnami's [minideb](https://github.com/bitnami/minideb)
+  a minimalist Debian-based container image that gives you a small base
+  container image and the familiarity of a leading Linux distribution.
+
+
+## Why not just contribute to Bitnami?
+
+We may eventually merge our work into Bitnami, for now we'd like to remain agile
+enough to build for platforms they don't (yet) support, like `arm64`, and to use
+supporting libraries (e.g. SASL) we've built that include fixes and features not
+yet in their mainstream counterparts.
```

Comment:
more likely the base distro

Reply:
"mainstream counterparts" sounds like we carry patches not present upstream (as opposed not fixed in the underlying distro). We try never to carry patches that upstream hasn't had a chance to at least review, except when it is urgent to do so.

But if you think most people will understand it as just having fixes not in the distro, feel free to keep.